Support QuickSpec

toshi0383 opened the pull request:
Support toggling between `Hoge.swift` and `HogeSpec.swift`.
I will rebase to master after #1 is merged.😉

toshi0383 committed:
Add support for QuickSpec

takasek commented on xcode_toggleImplAndTests.scpt:
```diff
@@ -16,17 +16,27 @@ on run
 		set destinationPath to do shell script command
+		if length of destinationPath < 1 then
+			set destinationName to (my replaceThis:(w1 & ".*") inString:sourceName usingThis:w3)
+            display dialog destinationName
```
Oh, 4-space tabs mixed!

takasek commented on xcode_toggleImplAndTests.scpt:
```diff
@@ -16,17 +16,27 @@ on run
 		# display dialog command
+		set command to "find " & quoted form of projectFolder & " -name " & quoted form of destinationName
 		set destinationPath to do shell script command
+		if length of destinationPath < 1 then
```
This logic is somewhat unstructural.
If you can, using array to compose possible `destinationNames` makes the logic simple.  What do you think?
ちなみに I had tried to use the array but it doesn't work fine, so I abandoned...

toshi0383 committed:
Remove display

toshi0383 commented:
Rebased to master and removed `display` lines.
Yea I think the logic is getting a little bit overwhelming. But I think we should move forward and merge anyways and leave refactoring as a future task. (I have to google it to figure out how to create an array in AppleScript!😇)

takasek commented:
>we should move forward and merge anyways and leave refactoring as a future task.

I got it. Exactly!
